scripts/generate_openapi_spec.py

```diff
@@ -89,9 +97,36 @@ def build_openapi_spec():
         click.echo(
             click.style('OpenAPI specification validated successfully',
                         fg='green'))
+        copy_openapi_specs(dev)
 
     return spec.to_dict()
 
 
+def copy_openapi_specs(dev):
+    """Copy generated and validated openapi specs to reana-commons module."""
+    if dev:
```

Comment:
we can move this if in `build_openapi_spec` not to enter the function if it's false